lib/Macros/Sources/SwiftMacros/DebugDescriptionMacro.swift

```diff
@@ -13,6 +13,7 @@
 import SwiftSyntax
 import SwiftSyntaxMacros
 import SwiftDiagnostics
+import _StringProcessing
```

Comment:
What's the story here? I don't think that this should be necessary...

Reply:
I added it to use `replacing(_:with:)` without importing `Foundation`. I've been under the impression that I couldn't (or shouldn't) import `Foundation` here. Is it permitted to import `Foundation` instead?

Reply:
I think it's fine for now, but _StringProcessing is implicitly imported normally.